addons/mail/static/tests/new/composer/composer_tests.js

```diff
@@ -212,6 +212,69 @@ QUnit.test(
     }
 );
 
+QUnit.test("keep emoji picker scroll value when re-opening it", async function (assert) {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["mail.channel"].create({ name: "roblox-carsurfing" });
+    const testScrollValue = 150;
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await click("button[aria-label='Emojis']");
+    document.querySelector(".o-mail-emoji-picker-content").scrollTop = testScrollValue;
+    await click("button[aria-label='Emojis']");
+    await click("button[aria-label='Emojis']");
+    assert.strictEqual(
+        document.querySelector(".o-mail-emoji-picker-content").scrollTop,
+        testScrollValue
+    );
+});
+
+QUnit.test("reset emoji picker scroll value after an emoji is picked", async function (assert) {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["mail.channel"].create({ name: "roblox-fingerskating" });
+    const testScrollValue = 150;
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await click("button[aria-label='Emojis']");
+    document.querySelector(".o-mail-emoji-picker-content").scrollTop = testScrollValue;
+    await click(".o-emoji:contains(😎)");
+    await click("button[aria-label='Emojis']");
+    assert.strictEqual(document.querySelector(".o-mail-emoji-picker-content").scrollTop, 0);
+});
+
+QUnit.test(
+    "keep emoji picker scroll value independent if two or more different emoji pickers are used",
+    async function (assert) {
+        const pyEnv = await startServer();
+        const channelId = pyEnv["mail.channel"].create({ name: "roblox-jaywalking" });
+        const testScrollValue = 150;
+        const testScrollValue2 = 200;
```

Comment:
I removed this variable. Better just hard-code the value: makes the test easier to read.